cmd/lockinfo-handlers.go

```diff
@@ -84,10 +89,57 @@ func getSystemLockState() (SystemLockState, error) {
 				LockType:    lockInfo.lType,
 				Status:      lockInfo.status,
 				Since:       lockInfo.since,
-				Duration:    time.Now().UTC().Sub(lockInfo.since),
+				Duration:    timeNow.Sub(lockInfo.since),
 			})
 		}
 		lockState.LocksInfoPerObject = append(lockState.LocksInfoPerObject, volLockInfo)
 	}
 	return lockState, nil
 }
+
+// filterLocks - Fetches locks held on bucket, matching prefix older than relTime.
```

Comment:
You don't need to say this as "filterLocks()"  - this is listLocks() - input params looks quite similar to listObjects(). relTime is like marker.